.circleci/config.yml

```diff
@@ -1,46 +1,36 @@
 version: 2.1
 
 orbs:
-  hmpps: ministryofjustice/hmpps@5.1.0
+  hmpps: ministryofjustice/hmpps@5.2.0
   slack: circleci/slack@4.4.2
   codecov: codecov/codecov@3.2.0
 
 parameters:
   alerts-slack-channel:
     type: string
-    default: farsight-alerts
+    default: cvl_alerts_security
 
   releases-slack-channel:
     type: string
-    default: farsight-alerts
+    default: the_elephants_devs
```

Comment:
bit torn about this, not a big fan of alerts going into channels that non-robots use